### ISSUE TITLE
Deploy docs failing

### ISSUE THREAD

**Issue by manavbhatia** [MASTmultiphysics/MAST3#31]
Travis for merge of PR #30 is [failing](https://travis-ci.com/github/MASTmultiphysics/MAST3/jobs/377947328) at the final stage of deploy docs with the following errors. 

```
rsync: delete_file: rmdir(.well-known/acme-challenge) failed: Permission denied (13)
rsync error: some files/attrs were not transferred (see previous errors) (code 23) at main.c(1196) [sender=3.1.2]

```

**Comment by jdeaton** [comment]
Sorry I haven't had a chance to get it taken care of. I search around on those errors really quickly and I think it has something to do with the HTTPS encryption on my web hosting or something potentially. The certificates just renewed for the first time the other day I saw in my email so I may need to do something there.

I'll try to get it taken care of ASAP.

**Comment by manavbhatia** [comment]
Thanks. 
Once fixed, I think we could just rerun the documentation test and it will fix the build error on the main site.

**Comment by jdeaton** [comment]
Closed in Pull Request #34
